dashboard/modules/state/state_head.py

```diff
@@ -114,6 +129,14 @@ async def list_objects(self, req) -> aiohttp.web.Response:
         data = await self._state_api.list_objects(option=self._options_from_req(req))
         return self._reply(success=True, message="", result=data)
 
+    @routes.get("/api/v0/runtime_envs")
+    @dashboard_optional_utils.aiohttp_cache
+    async def list_runtime_envs(self, req) -> aiohttp.web.Response:
```

Comment:
Does this list all runtime envs in the cluster?
Could you add some documentations?

Reply:
Can we also support listing by a special node_id?

Reply:
I will add a node id to the field

Reply:
We will support filter for (draft REP https://github.com/ray-project/enhancements/pull/8/files)

<img width="1597" alt="Screen Shot 2022-04-25 at 11 08 10 PM" src="https://user-images.githubusercontent.com/18510752/165233094-4ec849d6-dea6-433d-b7df-6640ea2b8708.png">

I will add node id and allow to filter entries by node id

Reply:
Actually, let's split out the node id to another PR (along with URI states). This is a bit complicated, and we need to think about the semantic.